### PR TITLE
Adding CI for notebooks 03, 04, 05 and 06

### PR DESCRIPTION
**Pull request by bschifferer**
Added CI with real data for notebook 03, 04, 05 and 06

### CODE REVIEW

File: examples/04-Exporting-ranking-models.ipynb [hunk 1]
```diff
@@ -70,9 +70,9 @@
      "name": "stderr",
```
Comment (rnyak) [inline examples/04-Exporting-ranking-models.ipynb:70]: <pre class="ql-syntax" spellcheck="false">Line #9.        train, valid = generate_data("aliccp-raw", int(NUM_ROWS), set_sizes=(0.7, 0.3))
</pre><p>is there an <code>else</code>  statement? what will happen if it is not True?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rnyak) [inline examples/04-Exporting-ranking-models.ipynb:70]: <pre class="ql-syntax" spellcheck="false">Line #1.    train_path = os.path.join(DATA_FOLDER, "train", "*.parquet")
</pre><p>Did you test that? <code>*.parquet</code> was not working for me what's why I had to do <code>"part.0.parquet"</code> instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (rnyak) [inline examples/04-Exporting-ranking-models.ipynb:70]: <pre class="ql-syntax" spellcheck="false">Line #2.    !apt-get install tree
</pre><p>Can you pls  remove these lines and instead add  <code>!pip install seedir</code>?   Seedir is a python package for creating, editing, and reading folder tree diagrams. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (rnyak) [inline examples/04-Exporting-ranking-models.ipynb:70]: <pre class="ql-syntax" spellcheck="false">Line #1.    !tree ./workflow
</pre><p>can you remove this line and  add the following instead?</p><pre class="ql-syntax" spellcheck="false">import seedir as sd
sd.seedir('workflow', style='lines', itemlimit=10, depthlimit=3, exclude_folders='.ipynb_checkpoints', sort=True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (bschifferer) [reply examples/04-Exporting-ranking-models.ipynb:70]: <p>The if statement writes data into the DATA_FOLDER. If it is not True, then it assumes there is already data in the folder (the raw data)</p>

Comment (bschifferer) [reply examples/04-Exporting-ranking-models.ipynb:70]: Yes the unittest passed and it worked for me

**Comment by bschifferer** [comment]
rerun tests

**Comment by bschifferer** [comment]
rerun tests

**Comment by bschifferer** [comment]
rerun tests

File: tests/tf/integration/test_ci_05_export_retrieval_model.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+from tests.conftest import REPO_ROOT
+
+
+@testbook(REPO_ROOT / "examples/05-Retrieval-Model.ipynb", timeout=180, execute=False)
+def test_func(tb):
+    tb.inject(
+        """
+        import os
+        os.environ["DATA_FOLDER"] = "/raid/data/aliccp/raw/"
```
Comment (rnyak) [inline tests/tf/integration/test_ci_05_export_retrieval_model.py:13]: is there a `raid` directory with raw alicpp dataset on CI machine?

**Review by rnyak — APPROVED** [review]

File: tests/tf/integration/test_ci_05_export_retrieval_model.py (hunk 2 above)
Comment (bschifferer) [reply tests/tf/integration/test_ci_05_export_retrieval_model.py:13]: Yes should be in my understanding

**Comment by bschifferer** [comment]
@benfred @jperez999 @marcromeyn - it seems Jenkins executed all the CI notebook tests. This was not the case for the PR, where I submitted the first CI notebook tests. In my understanding, Jenkins should not execute them. Do you know why this is the case?

**Comment by bschifferer** [comment]
rerun tests

**Comment by bschifferer** [comment]
@rnyak I added metadata and container info - can you review the changes, please?

**Pull request merged**